matter.py

```diff
@@ -557,6 +592,19 @@ def do_patch_grub_cfg_icons():
         # eventually it will be solved with an autoremove
         exit(0)
 
+
+def do_patch_grub_cfg_icons(icons=None):
+    if icons is None:
+        icons = argparse.icons
+
+    info(f"Begin {GRUB_CFG_PATH} patch")
+    with open(GRUB_CFG_PATH, "r", newline="") as f:
```

Comment:
(This is now at line 583)
Within this function `do_patch_grub_cfg_icons()`, `grub_cfg` is needed to build the new grub cfg later. I thought it was a little less messy to put this within `get_entry_names()` because it was repeated quite a lot of times throughout the script.